### PR TITLE
Migrating to poetry

### PR DESCRIPTION
**Pull request by GirZ0n**
**Changes**
- Migrated from the Paddle build system to Poetry

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+[tool.poetry]
+name = "hyperstyle-analysis-prod"
+version = "0.1.0"
+description = "Repository containing several tools-helpers to work with data from the Hyperstyle platform"
+authors = ["Anastasia Birillo <anastasia.birillo@jetbrains.com>"]
```
Comment (nbirillo) [inline pyproject.toml]: Please, put yourself also to the authors list

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    { include = "data_collection" },
+    { include = "data_labelling" },
+    { include = "jba" },
+    { include = "preprocessing" },
+    { include = "templates" },
+]
+
+# Core dependencies
+[tool.poetry.dependencies]
+python = "~3.9"
```
Comment (nbirillo) [inline pyproject.toml:22]: I'd suggest to use a hard version of Python to be sure about the same environment everywhere

**Review by nbirillo — CHANGES_REQUESTED** [review]
Thanks a lot!

File: pyproject.toml (hunk 2 above)
Comment (GirZ0n) [reply pyproject.toml:22]: Poetry cannot install the specified Python version, so you must do it manually. Therefore I used the tilde requirement here

File: pyproject.toml (hunk 1 above)
Comment (GirZ0n) [reply pyproject.toml]: I also added Maria Tigina here :-)

**Review by nbirillo — APPROVED** [review]

**Pull request merged**